### DISCUSSION TITLE
"grpc_message":"Received http2 header with status: 503"

### DISCUSSION THREAD

**Discussion by sativiensiri** [milvus-io/milvus#30019] [Q&A and General discussion]
Hello Team,

I successfully installed Milvus 2.3.4 standalone on AWS EC2 with attu 2.2.6, it's working well. But once try to connect with pymilvus==2.2.0 , it's can't and have error as below details. Anyone has experiences, please suggestion.

Thanks

Details:
Traceback (most recent call last):
  File "C:\Users\25407\PycharmProjects\milvusDB\venv\lib\site-packages\pymilvus\decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "C:\Users\25407\PycharmProjects\milvusDB\venv\lib\site-packages\pymilvus\client\grpc_handler.py", line 236, in has_collection
    reply = rf.result()
  File "C:\Users\25407\PycharmProjects\milvusDB\venv\lib\site-packages\grpc\_channel.py", line 744, in result
    raise self
grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Received http2 header with status: 503"
	debug_error_string = "{"created":"@1705379873.084000000","description":"Error received from peer ipv4:13.215.254.142:19530","file":"src/core/lib/surface/call.cc","file_line":967,"grpc_message":"Received http2 header with status: 503","grpc_status":14}"
>

**Comment by yhmo** [comment]
Use the recommended pymilvus version according to this matrix:
https://github.com/milvus-io/pymilvus?tab=readme-ov-file#compatibility

Usually, sdk v2.2.x is for milvus v2.2.x, sdk v2.3.x is for milvus v2.3.x. The same with Attu.

**Reply by sativiensiri** [reply]
Thank you for you recommendation, I will looking on it.

**Reply by sativiensiri** [reply]
After try with sdk v2.3.4, it's still having the same error.

**Reply by yhmo** [reply]
The milvus server might fail to start, check the log.
